rfcs/0109-nixpkgs-generated-code-policy.md

```diff
@@ -0,0 +1,210 @@
+---
+feature: nixpkgs-generated-code-policy
+start-date: 2021-10-12
+author: John Ericson (@Ericson2314)
+co-authors: (find a buddy later to help out with the RFC)
+shepherd-team: @L-as @sternenseemann @tomberek @DavHau
+shepherd-leader: @sternenseemann
```

Comment:
interpreting @KFearsoff's comment as accepting the nomination...

>@lheckemann proposed me as a shepherd, so I could try doing that. Not tomorrow, but after that.

```suggestion
shepherd-team: @L-as @kfearsoff @tomberek @DavHau
shepherd-leader: @kfearsoff
```

Reply:
Then I'd take over for @Ericson2314 at the same time?

Reply:
Woah there, haven't agreed to be the shepherd leader yet. Need to understand the responsibilities first, which I don't understand too well right now.

Reply:
I had the same reaction when I was nominated as shepherd chair for rfc133... and had zero prior involvement in the rfc process.

It really just means being the person who's responsible for scheduling meetings, taking notes, and not dropping the ball (which I did, a few times).

Maybe "shepherd secretary" would be a bit less scary to people getting roped in for the first time.

Reply:
Understandable, sorry about that! We'll try to take previous experience into account in future shepherd team decisions. Tom has volunteered to take the lead on this one.

```suggestion
shepherd-team: @L-as @kfearsoff @tomberek @DavHau
shepherd-leader: @tomberek
```

Reply:
I'm still unclear on why I'm being removed altogether?